pkg/moov/undewriting_models.go

```diff
@@ -36,7 +36,7 @@ type VolumeByCustomerType struct {
 
 type CardVolumeDistribution struct {
 	EcommercePercentage     int32 `json:"ecommercePercentage"`
-	CardPresentPercentage   int32 `json:"cardPresentPercentagePercentage"`
+	CardPresentPercentage   int32 `json:"cardPresentPercentage"`
```

Comment:
Looks like this made its way into platform-api so some customers may depend on it? Changing it would result in a breakage and requires we notify. 

https://github.com/search?q=repo%3Amoovfinancial%2Fplatform-api%20cardPresentPercentagePercentage&type=code

Reply:
Since we didn't have customers using it we made the change. This one is ready for review. Tests are failing but from another issue that appears to have started failing tests since yesterday.

Reply:
We'll get the tests fixed up. I thought I saw failures related to this since the server is returning `cardPresentPercentagePercentage`, which is a chicken and egg problem. How close are we to the server fix being deployed?

Reply:
https://github.com/moovfinancial/moov-go/pull/121 has fixed the failures seen here, so I'm going to merge this and see if everything passes.

Reply:
Its been deployed. The first run earlier was failing because the server changes hadn't been deployed yet. Now the runs are failing for the other tests I mentioned.